.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "root": true,
+  "env": {
+    "browser": true
+  },
+  "extends": [
+    "airbnb",
+    "plugin:prettier/recommended"
+  ],
+  "plugins": [
+    "prettier"
+  ],
+  "rules": {
+    "prettier/prettier": "error"
```

Comment:
This is what `plugin:prettier/recommended` also does. Do we need both?
👉  https://github.com/prettier/eslint-plugin-prettier/blob/master/eslint-plugin-prettier.js#L113

Since we're using this together with airbnb, there is also [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier/blob/master/README.md), which selectively deactivates rules that would conflict with prettiers whitespace choices.

Reply:
No, I think not, good catch! 👍